Makefile

```diff
@@ -50,70 +49,61 @@ autoloads:
 
 # Create .tgz and .zip files only
 # GNUTAR=gtar make tarballs
-tarballs: $(ESSDIR)
+.PHONY: rel-tarballs
+rel-tarballs: rel-staging
 	@echo "**********************************************************"
 	@echo "** Making distribution of ESS for (pre)release $(ESSVERSION) from $(ESSDIR)/"
-	@echo "** Making pdf and html documentation"
-	cd $(ESSDIR)/doc/ ; $(MAKE) pdf
-	cd $(ESSDIR)/doc/ ; $(MAKE) html
-	@echo "** Creating .tgz file **"
 	test -f $(ESSDIR).tgz && rm -rf $(ESSDIR).tgz || true
+	grep -E 'defvar ess-(version|revision)' $(ESSDIR)/lisp/ess-custom.el
+	@echo "** Creating .tgz file **"
 	$(GNUTAR) hcvofz $(ESSDIR).tgz $(ESSDIR)
 	@echo "Signing tgz file"
-	$(GPG) -ba -o $(ESSDIR).tgz.sig $(ESSDIR).tgz
+	-$(GPG) -ba -o $(ESSDIR).tgz.sig $(ESSDIR).tgz
```

Comment:
What's the difference between `-$` and `$`?

Reply:
Prevent gpg from failing the recipe.
Travis doesn't have a gpg key, and I don't want travis to fail.

Reply:
But that means the local GPG error can be missed as well, right?

Reply:
Fixed